app/views/teams/index.html.erb

```diff
@@ -1,21 +1,37 @@
 <% content_for :title_bar do %>
   <h1 class="top-bar-title"><%= t('teams.switch') %></h1>
 <% end %>
-<div class="row">
-  <div class="col-xs-12 col-sm-6">
-    <div class="form-wrapper">
-      <%= form_tag(switch_teams_path, method: :get) do %>
-        <div class="field-wrapper">
-          <%= label_tag :team_slug, 'Teams', class: 'col-sm-2 control-label' %>
-          <div class="col-sm-10">
-            <%= select_tag :team_slug, options_from_collection_for_select(@teams, :slug, :name), disabled: @teams.blank? %>
+
+<div class="col-md-8 col-md-offset-2">
```

Comment:
Shouldn't this column be inside of a row? It may break because of the negative margin Bootstrap adds to cols.